src/test/rustdoc-ui/intra-doc/errors.stderr

```diff
@@ -134,10 +142,12 @@ error: unresolved link to `m`
   --> $DIR/errors.rs:98:6
    |
 LL | /// [m()]
-   |      ^^^
-   |      |
-   |      this link resolves to the macro `m`, which is not in the value namespace
-   |      help: to link to the macro, add an exclamation mark: `m!`
+   |      ^^^ this link resolves to the macro `m`, which is not in the value namespace
+   |
+help: to link to the macro, add an exclamation mark
+   |
+LL | /// [m!()]
+   |       ^
```

Comment:
For this case the replacement span was pointing at the point immediately after `m`, but it needs to start there and end at the current `sp.hi()`.

Reply:
Somehow, even after all the other cases have been fixed, this one hasn't 🤔